chat-plugins/poll.js

```diff
@@ -0,0 +1,155 @@
+/*
+* Poll chat plugin
+* This plugin allows roomauth (default: moderator and above) to run a poll in a room.
+* Every room can have one poll, and every user can vote. The results are displayed for
+* users that have voted, and are updated in real time. The poll can be closed with /endpoll.
+* By bumbadadabum with (a lot of) help from Zarel.
+*/
+
+var Poll = (function () {
+    function Poll(room, question, options) {
+        this.room = room;
+        this.question = question;
+        this.voters = [];
+        this.totalVotes = 0;
+
+        this.options = [];
+        for (var i in options) {
+            this.options.push({name: options[i], votes: 0});
+        }
+    }
+
+    Poll.prototype.vote = function(user, option) {
```

Comment:
I might be wrong, but isn't the more normal "form" or whatever on this repository `function (option1, option2)` instead of no space between `function` and the opening parent?